plugins/git-town/README.md

```diff
@@ -0,0 +1,22 @@
+# Git-Town plugin
+
+This plugin adds completion and aliases for the `git-town` command. More information
+at https://www.git-town.com.
+
+Enable git-town plugin in your zshrc file:
+```
+plugins=(... git-town)
+```
+
+## Aliases
+
+More information about `git-town` commands:
+https://github.com/Originate/git-town#commands
+
+| Alias   | Command                    | Description                            |
+|---------|----------------------------|----------------------------------------|
+| `gt`    | `git town`                 | Git-Town command                       |
+| `gth`   | `git town hack`            | New feature branch off main branch     |
+| `gtsy`  | `git town sync`            | Updates current branch w/ all changes  |
+| `gtnpr` | `git town new-pull-request`| Create a new pull request              |
+| `gtsh`  | `git town ship`            | Delivers feature branch and removes it |
```

Comment:
What about adding `gtc` for `git town continue`? I use that all the time.

Reply:
@kevgo added `continue` and another frequently used `abort`.  

Also added the full list of commands to the PR in case others have suggestions going forward.